cluster-autoscaler/cloudprovider/aws/README.md

```diff
@@ -31,6 +31,7 @@ The following policy provides the minimum privileges necessary for Cluster Autos
         "autoscaling:DescribeAutoScalingGroups",
         "autoscaling:DescribeAutoScalingInstances",
         "autoscaling:DescribeLaunchConfigurations",
+        "autoscaling:DescribeTags",
```

Comment:
If you don't have permission, this error will occur
```
E0107 12:41:18.399807       1 aws_manager.go:265] Failed to regenerate ASG cache: cannot autodiscover ASGs: AccessDenied: User: arn:aws:sts::030787254034:assumed-role/tools-kube-worker-dev-role/i-00e02057dcf6619ae is not authorized to perform: autoscaling:DescribeTags because no identity-based policy allows the autoscaling:DescribeTags action
	status code: 403, request id: afe418c9-099d-474c-a302-961b2247697d
F0107 12:41:18.399830       1 aws_cloud_provider.go:389] Failed to create AWS Manager: cannot autodiscover ASGs: AccessDenied: User: arn:aws:sts::030787254034:assumed-role/tools-kube-worker-dev-role/i-00e02057dcf6619ae is not authorized to perform: autoscaling:DescribeTags because no identity-based policy allows the autoscaling:DescribeTags action
	status code: 403, request id: afe418c9-099d-474c-a302-961b2247697d
```